Source/Tools/FEXLoader/AOT/AOTGenerator.cpp

```diff
@@ -103,7 +104,7 @@ void AOTGenSection(FEXCore::Context::Context* CTX, ELFCodeLoader::LoadedSection&
 
   // This code is tricky to refactor so it doesn't allocate memory through glibc.
   FEXCore::Allocator::YesIKnowImNotSupposedToUseTheGlibcAllocator glibc;
-  for (int i = 0; i < FEXCore::CPUInfo::CalculateNumberOfCPUs(); i++) {
+  for (int i = 0; i < FEX::CPUInfo::CalculateNumberOfCPUs(); i++) {
```

Comment:
Should have caught this in the previous review, but perhaps the retrieval of the upper limit should be stored into a variable and then used in the limit, so that it's not doing all of the string processing over and over for every iteration

Reply:
Good shout, moved it to a temp variable.